okteto/config.yaml

```diff
@@ -0,0 +1,39 @@
+backend:
```

Comment:
This file won't work as it is, you can download like 
this -> https://raw.githubusercontent.com/civo/kubernetes-marketplace/master/aspnet/manifest.yaml
then use it in install.sh